lib/train/extras/command_wrapper.rb

```diff
@@ -114,15 +113,22 @@ def run(script)
       # especially in local mode, we cannot be sure that we get a Powershell
       # we may just get a `cmd`.
       # TODO: we may want to opt for powershell.exe -command instead of `encodeCommand`
-      "powershell -encodedCommand #{WinRM::PowershellScript.new(safe_script(script)).encoded}"
+      "powershell -encodedCommand #{encoded(safe_script(script))}"
     end
 
-    # reused from https://github.com/WinRb/WinRM/blob/master/lib/winrm/command_executor.rb
     # suppress the progress stream from leaking to stderr
     def safe_script(script)
       "$ProgressPreference='SilentlyContinue';" + script
     end
 
+    # Encodes the script so that it can be passed to the PowerShell
+    # --EncodedCommand argument.
+    # @return [String] The UTF-16LE base64 encoded script
+    def encoded(script)
```

Comment:
Yeah. Also, there is no `PowershellScript` class in winrm v2 so we cant leverage that.